### ISSUE TITLE
coolprop with spring boot

### ISSUE THREAD

**Issue by Shilei-Cooper** [CoolProp/CoolProp#1986]
hi all:
I want to use coolprop with java in spring boot. 
but here is a big problem that the deamApplication must not in the default package but the coolprop.java must in the default package so that it can link to dll.
when I put the file .java in a package, an error occurred with "java.lang.UnsatisfiedLinkError: com.cup.CoolPropJNI.AbstractState_factory__SWIG_0"
I use the  command {#code -DCOOLPROP_SWIG_OPTIONS="-package com.cup" } to appoint the path but it didn't worked.
how can i deal with this f**k problem?? maybe some nice guy can help make a new dll with the package com.cup.
best wish for all

**Comment by ibell** [comment]
Can you please post the complete build log from the shell?  Might be informative.

**Comment by Shilei-Cooper** [comment]
It doesn't help beacuse it's a run time exception.

**Comment by ibell** [comment]
But maybe the warnings in the output hint at why you might get runtime exception after you build :)

**Comment by Shilei-Cooper** [comment]
yeah i kown why, because java link it with a package name ,like "java.lang.UnsatisfiedLinkError: com.cup.CoolPropJNI.AbstractState_factory__SWIG_0" , and if i drop the package name "com.cup.", it can work .
but i really need the package name. so how can i get a .dll with a package name ?   -.-

**Comment by ibell** [comment]
You can just rename the DLL file?

>

**Comment by ibell** [comment]
Closing as part of an issue-tracker cleanup — this issue has had no activity in over two years.

*To be transparent: this comment and the close action are being performed by Claude (Anthropic's AI coding assistant), running on behalf of @ibell. Ian reviewed and approved the bulk-close criteria; I am the one executing the action.*

If you would like this issue reconsidered, please feel free to **re-open it** — a fresh comment with current behaviour against the latest CoolProp release is the most useful way to revive the discussion.

— Claude (acting for @ibell)
